charts/cluster/values.schema.json

```diff
@@ -197,7 +197,29 @@
                     "type": "array"
                 },
                 "initdb": {
-                    "type": "object"
+                    "type": "object",
+                    "properties": {
+                        "database": {
+                            "type": "string"
+                        },
+                        "owner": {
+                            "type": "string"
+                        },
+                        "secret": {
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "type": "string"
+                                }
+                            }
+                        },
+                        "postInitSQL": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            }
+                        }
+                    }
```

Comment:
This bit isn't correct as there are many more options possible. Can you remove the changes here and leave the loose validation instead?

Reply:
Sure!